disputils/confirmation.py

```diff
@@ -88,3 +92,23 @@ async def confirm(self, text: str, user: discord.User = None, channel: discord.T
             channel = self._ctx.channel
 
         return await super().confirm(text, user, channel)
+
+    async def text_confirmation(
+        self, text: str, color: discord.Color, author: discord.User, show_author: bool, channel: discord.Channel, timeout: int = None
+    ) -> bool:
+        def input_check(msg: Message) -> bool:
+            return msg.author == self._ctx.author and msg.channel == self._ctx.channel
+
+        text += "Type `YES` to Accept, Else Decline."
```

Comment:
So, What do we do then, how can i improve code? so It's better?

Reply:
Well, just remove the hard-coded string I guess. There already is a `text` parameter so there is no real need to add anything to that text. I think that'd be the simplest solution at least.
Let me know what you think.

Reply:
Ah! Sure Sorry for the later reply, if you're on Discord, We can talk, and decide on working on this together, since i am lot interested to contribute.

Reply:
He has discord \OwO/ --> LiBa01#8817 (I'm his personal stalker). You don't have to thank me UwU

Reply:
Lmao, it's cool, thanks :D what's your discord?

Reply:
Enceladus#0815